dependency_licenses/FREETYPE2.txt

```diff
@@ -0,0 +1,35 @@
+The  FreeType 2  font  engine is  copyrighted  work and  cannot be  used
+legally  without a  software license.   In  order to  make this  project
+usable  to a vast  majority of  developers, we  distribute it  under two
+mutually exclusive open-source licenses.
+
+This means  that *you* must choose  *one* of the  two licenses described
+below, then obey  all its terms and conditions when  using FreeType 2 in
+any of your projects or products.
+
+  - The FreeType License, found in  the file `FTL.TXT', which is similar
```

Comment:
Should the files `FTL.TXT` and `GPLv2.TXT` referenced by FreeType be included as well?

Reply:
I don't know - the reader is instructed to pick one of them, so if we include both, then we're including a license that doesn't necessarily apply... which I think could be confusing.

Reply:
It should be easy enough for people to find those files if they need them, and let's keep things simpler. We can add them if necessary later.